service/history/replication/stream_receiver.go

```diff
@@ -162,15 +161,15 @@ func (r *StreamReceiverImpl) sendEventLoop() {
 			timer.Reset(r.Config.ReplicationStreamSyncStatusDuration())
 			if err := r.ackMessage(r.stream); err != nil {
 				r.logger.Error("StreamReceiver exit send loop", tag.Error(err))
```

Comment:
since fun returns error now, do we still need to log the error here? i would just let caller decide what to do the error

Reply:
That's right. I think in this code path, we have many placed logging the error. Will consolidate them in next revision.